### ISSUE TITLE
Summary - Detail

### ISSUE THREAD

**Issue by PeterLappo** [jupyter/help#371]
Hi,
I have some web documents that i've loaded into elasticsearch which I want to search and then display the pages (pages are html, xml, doc, pdf, image etc). I wrote a basic notebook that allowed me to do this but it displays all results including the pages some of which are quite large in display terms. So far so good. But what I'd like to do is click on a summary row and in a separate panel display the page contents in scrollable window and when another row is clicked new contect appears. Elasticsearch stores the web documents in base64 and can't display them directly. But Jupiter has the necessary libraries to decode and display the base64 but as a consequence the data is not a simple URL.

I'm a Jupyter beginner so don't know what is possible and haven't found any suitable examples to work from so I'm not sure if its possible to do what I want without a good deal of programming (which I'm not adverse to). I've also looked at JupyterLab and Beakerx neither of which appeared to offer a simple solution.

Basically I want something like callback on a table cell that displays its data in a separate window.

Does anyone have any ideas / pointers?

Thanks 
Peter
